README.md

```diff
@@ -200,6 +200,8 @@ option unset.
   -grpc-port port
         grpc server port. Can be in the form of host:port, ip:port or port or
 /unix/domain/path or "disabled" to not start the grpc server. (default "8079")
+  -grpc-metadata value
+        Metadata that will be added to the grpc request. e.g. -grpc-metadata "key: value".
```

Comment:
Can we just use the existing -H ?

Reply:
Sorry for the late reply, I recently got infected with covid-19. grpc metadata is equal to http header most of the time but there are some nuances (like not allowing to set some reserved headers etc.), reusing -H might cause some confusion? I don't know, it's up to you to decide whether to change it or not, I'm ok with both.

Reply:
I think if we intend to reuse `-H`, we can add more clearer flag doc.

Reply:
sure we can update the doc to say Headers or GRPC meta data but we already have way too many flags so I'm relunctant to add more that are same (assuming, correct me as I don't know, that grpc meta data has the same key:value pair string:string semantic) ?

Reply:
done.

Reply:
Btw sorry to hear - hope you're recovering fully/quickly! no rush and thanks for having done all this despite being sick